master/usage/ipv6.md

```diff
@@ -1,31 +1,158 @@
 ---
-title: IPv6 Support
+title: Enabling IPv6 Support
 canonical_url: 'https://docs.projectcalico.org/v3.0/usage/ipv6'
 ---
 
-{{site.prodname}} supports connectivity over IPv6, between compute hosts, and
-between compute hosts and their VMs. This means that, subject to
-security configuration, a VM can initiate an IPv6 connection to another
-VM, or to an IPv6 destination outside the data center; and that a VM can
-terminate an IPv6 connection from outside.
-
-## Requirements for containers
-
-Containers have no specific requirements for utilizing IPv6
-connectivity.
-
-## Requirements for guest VM images
+### About IPv6 Support
+
+You may be interested in this document if you want to enable IPv6
+Support. This would be because:
+- You want workloads to communicate over IPv6.
+- You want workloads to initiate connections to IPv6 services.
+- You want workloads to terminate incomint IPv6 connections.
+
+### Purpose of this page
+
+Provide guidance on how to enable IPv6 addressing for Calico networked
+workloads.
+
+## Enabling IPv6 with Kubernetes
+
+The sections below are not intended to be a detailed setup guide but
+to layout the settings necessary to enable IPv6 operation of Kubernetes
+and Calico.
+
+### Prerequisites
+
+#### Host Prerequisites
```

Comment:
We are trying to standardize on sentence case for the headings, so "Prerequisites" should be lowercased -> "prerequisites"